database/migrations/0037-community_federation_tables.ts

```diff
@@ -0,0 +1,53 @@
+/* MIGRATION TO CREATE THE FEDERATION COMMUNITY TABLES
+ *
+ * This migration creates the `community` and 'communityfederation' tables in the `apollo` database (`gradido_community`).
+ */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`community\` (
+      \`id\` int unsigned NOT NULL AUTO_INCREMENT,
+      \`uuid\` varchar(37) NOT NULL,
+      \`name\` varchar(255),
+      \`description\` varchar(255),
+      \`createdAt\` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+      PRIMARY KEY (\`id\`),
+      UNIQUE KEY \`uuid\` (\`uuid\`)
+    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+  `)
+  await queryFn(`
+    CREATE TABLE IF NOT EXISTS \`community_federation\` (
+      \`id\` int unsigned NOT NULL AUTO_INCREMENT,
+      \`uuid\` varchar(36) NOT NULL,
+      \`foreign\` tinyint DEFAULT '0',
```

Comment:
As stated above. When `true` I understand to have a foreign community. Change to `self` (if not reserved)